src/libs/migrations/RenameExpensifyNewsStatus.ts

```diff
@@ -15,8 +14,9 @@ export default function () {
                 Onyx.disconnect(connectionID);
 
                 // Fail early here because there is nothing to migrate
-                if (!user || _.isNull(user.expensifyNewsStatus) || _.isUndefined(user.expensifyNewsStatus)) {
+                if (user?.expensifyNewsStatus === null || user?.expensifyNewsStatus === undefined) {
                     Log.info('[Migrate Onyx] Skipped migration RenameExpensifyNewsStatus');
+                    // @ts-expect-error In that case we don't need to resolve anything
```

Comment:
Instead of adding this, you can just change `return new Promise(...` to `return new Promise<void>(...` up there.